### ISSUE TITLE
trigonometric functions

### ISSUE THREAD

**Issue by jay-dave** [lanl-ansi/Alpine.jl#125]
Are there any plans to support trigonometric functions in the coming months?

**Comment by harshangrjn** [comment]
Yes! We do have this in the pipeline. Current efforts are towards migrating to be compatible with JuMP 0.19+. Once done with this, we plan to add relaxation models for transcendental functions.

**Comment by krishpat** [comment]
I was going to ask this too. Trig function support will be helpful. Also, support for functions with fractional exponents will be useful since they seem to appear in many applications. Thanks!

**Comment by harshangrjn** [comment]
#28 #29 Tagging related enhancements!
